ClimbingGearBackend/Controllers/UserGearController.cs

```diff
@@ -1,32 +1,31 @@
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
 using ClimbingGearBackend.Models;
-using ClimbingGearBackend.Infrastucture;
+using ClimbingGearBackend.Interfaces;
 
 namespace ClimbingGearBackend.Controllers
 {
   [Route("api/[controller]")]
   [ApiController]
-  //TODO:#1 Enable Authorization and have DB querys only return logged in UserGear
+  //TODO:#1 Enable Authorization and have DB querys only return logged in User Gear
```

Comment:
Shouldn't this be TODO #5? Also, I don't know if VS Code parses the format you have, but if you do `TODO(#5): xyz` then VS Code can actually parse out the ticket number and give you a summary of the ticket in the editor itself. The parenthetical format is also industry standard.

Reply:
>VS Code can actually parse out the ticket number and give you a summary of the ticket in the editor itself. 
Hmm, My Vscode doesn't parse the issue ticket, Is that an extenstion?
> Shouldn't this be TODO #5?
Yes, changed.

Reply:
It might be the GitHub extension? But I think that is built in. Or maybe it depends on the language? *shrug*